src/kibana/plugins/vis_types/controls/radius_ratio_option.html

```diff
@@ -0,0 +1,16 @@
+<div>
+  <span class="hintbox_label" ng-click="showDotRatioHint = !showDotRatioHint">
+    <label>
+      Dot Size Ratio:
+      <kbn-info
+        placement="right"
+        class="text-info"
```

Comment:
If we are going to change the color of one kbn-info element we should probably change it for all.